replace_unix.go

```diff
@@ -6,21 +6,24 @@ import (
 	"syscall"
 )
 
+func mprotectCrossPage(addr uintptr, len int, prot int) {
```

Comment:
please change the `len` variable name to something else, as you're shadowing the builtin

Reply:
ok, 3q for reviewing